OpenSim/Simulation/Model/Frame.cpp

```diff
@@ -180,12 +180,42 @@ SimTK::Vec3 Frame::expressVectorInAnotherFrame(const SimTK::State& state,
     return findTransformBetween(state, frame).R()*vec;
 }
 
-SimTK::Vec3 Frame::findLocationInAnotherFrame(const SimTK::State& state, const
-        SimTK::Vec3& point, const Frame& otherFrame) const
+SimTK::Vec3 Frame::expressVectorInGround(const SimTK::State& state,
+                                const SimTK::Vec3& vec_F) const
+{
+    return getTransformInGround(state).R()*vec_F;
```

Comment:
The two functions here are meant to perform slightly different operations. `expressVectorInGround` is supposed to only express a `Vec3` in the ground basis, whereas `findLocationInGround` is supposed to treat the `Vec3` as a point and both express the `Vec3` in the ground basis and perform a translation if needed. The documentation [here](https://simtk.org/api_docs/simbody/3.5/singletonSimTK_1_1Transform__.html#a6431752e53d0aa5556082a0282a8a214) indicates that the latter occurs when a `Transform` * `Vec3` occurs. Is this redundant in the case when vectors are transformed to Ground?